providers/flagd/src/main/resources/flags.json

```diff
@@ -1,184 +0,0 @@
-{
-  "$id": "https://flagd.dev/schema/v0/flags.json",
```

Comment:
I pushed a small change to some test files that use this schema; they now use the pulled/gitignored one.

The test files using this schema only use it for editor support (so editing the schemas locally has editor support) not in actual unit tests.

Reply:
Ahh, there they have been used. I was wondering why there have been JSON schemas checked in in resources when they're copied to a different resource directory on build.